simd/keccakf1600/example_test.go

```diff
@@ -72,8 +70,9 @@ func Example() {
 		}
 	}
 
-	fmt.Printf("%x\n%x\n%x\n%x\n", hashes[0], hashes[1], hashes[2], hashes[3])
-	// Output: 9b48efc4f4e562fe28c510b2ad3966b101ac20066dc88117d85a595cc965f7e4
+	fmt.Printf("\n%x\n%x\n%x\n%x\n", hashes[0], hashes[1], hashes[2], hashes[3])
+	// Output:
```

Comment:
This "Output:" comment is read by the Go testing framework and this will break the test.  (See [testable examples on the Go blog.](https://blog.golang.org/examples))

Reply:
That's why I added and extra '\n' both in the comment and in the printf.

Reply:
Oh, sorry, missed that.